funnel/forms/participant.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+import os
+import wtforms
+import baseframe.forms as forms
+from baseframe import __
+from wtforms.ext.sqlalchemy.fields import QuerySelectMultipleField
+from wtforms.widgets import CheckboxInput, ListWidget
+
+__all__ = ['ParticipantForm', 'ParticipantBadgeForm']
+
+
+class ParticipantForm(forms.Form):
+    fullname = forms.StringField(__("Full Name"), validators=[forms.validators.DataRequired()])
+    email = forms.EmailField(__("Email"), validators=[forms.validators.DataRequired(), forms.validators.Length(max=80)])
+    phone = forms.StringField(__("Phone number"), validators=[forms.validators.Length(max=80)])
+    city = forms.StringField(__("City"), validators=[forms.validators.Length(max=80)])
+    company = forms.StringField(__("Company"), validators=[forms.validators.Length(max=80)])
+    job_title = forms.StringField(__("Job Title"), validators=[forms.validators.Length(max=80)])
+    twitter = forms.StringField(__("Twitter"), validators=[forms.validators.Length(max=15)])
+    events = QuerySelectMultipleField(__("Events"),
```

Comment:
Because of the concerns in hasgeek/baseframe#112, I'd move this context into this file itself if possible.